Sentinel/Classes/Core/UIImage+Resize.swift

```diff
@@ -7,10 +7,20 @@
 
 import Foundation
 extension UIImage {
-    func resize(for size: CGSize = CGSize(width: 24, height: 24)) -> UIImage? {
+
+    static func loadImageFromBundle(name: String) -> UIImage? {
+        let frameworkBundle = Bundle(for: Sentinel.self)
+        guard let bundleURL = frameworkBundle.resourceURL?.appendingPathComponent("Sentinel.bundle")
+        else { return nil }
+        let resourceBundle = Bundle(url: bundleURL)
+        guard let image = UIImage(named: name, in: resourceBundle, compatibleWith: nil) else {return nil}
+        return image
+    }
+
+    static func resize(for size: CGSize = CGSize(width: 24, height: 24), _ image: UIImage?) -> UIImage? {
```

Comment:
Why do we need to do resizing it the app, can we just add the images of correct size to the app.

Reply:
We added images from Android. They only had XML images and we couldn't use them like that, so we used a pdf converted we found online to get a format that we can use. It got messed up during the process and now we have to resize them 😞

Reply:
Can you send me the original images in slack, I'll edit them 😉